code/modules/mob/living/deity/phenomena/starlight.dm

```diff
@@ -70,7 +70,7 @@
 			extension.expected_helmet = helm.type //We only do by type because A. its easier to manage and B the chances of it being non-unique in a normal game is very small
 		if(form["weapon"])
 			var/w_type = form["weapon"]
-			L.equip_to_slot_or_store_or_drop(new w_type(T), slot_r_hand)
+			L.put_in_hands(new w_type(T))
```

Comment:
I'll be honest; I've been glossing over the mass-replace stuff. But in some cases change of function does matter somewhat: e.g. for the antag item ones it's somewhat relevant that they are stored prior to dropped. May be worth checking a bit more (not really pointing to this one in particular; I can't tell from the context if it matters here or even what mob type this is).

Reply:
I've added a `put_in_hands_or_store_or_drop` and will go through and make sure I swapped the right procs out for it.